browser/src/canvas/sections/CommentSection.ts

```diff
@@ -1540,14 +1540,18 @@ export class Comment extends CanvasSectionObject {
 		if (this.sectionProperties.docLayer._docType === 'spreadsheet')
 			return;
 
+		var innerText;
 		if (this.isEdit())
-			this.sectionProperties.collapsedInfoNode.innerText = '!';
+			innerText = '!';
 		else if (replycount === '!' || typeof replycount === "number" && replycount > 0)
-			this.sectionProperties.collapsedInfoNode.innerText = replycount;
+			innerText = replycount;
 		else
-			this.sectionProperties.collapsedInfoNode.innerText = '';
+			innerText = '';
 
-		if (this.sectionProperties.collapsedInfoNode.innerText === '' || this.isContainerVisible())
+		if (this.sectionProperties.collapsedInfoNode.innerText != innerText)
```

Comment:
Minor point: comparisons with the `innerText` in this function are without coercion (i.e. `===` and `!==`). Shouldn't this too do the same?

Incidentally, `===` and `!==` are slightly faster than `==` and `!=` (for the obvious reason that type coercion needs to be considered/tried for the latter, where the former will short-circuit if the types aren't the same).

Reply:
Sounds reasonable - but doesn't work: we inject numbers into innerText which are then converted to strings internally, and when we get them back we get to do a sloppy compare here I think. Either way - it's a big win for rather a minimal change.

If you'd like to do a cleanup, and re-validate, feel free but ... =)